.github/workflows/buck2-ci.yml

```diff
@@ -24,3 +24,19 @@ jobs:
         run: script/buck2 test //... -- --env PERCY_TOKEN=$PERCY_TOKEN
         env:
           PERCY_TOKEN: '${{ secrets.PERCY_TOKEN }}'
+
+      - name: buck2 elm-format (with comments)
+        if: github.event.pull_request
+        run: |
+          set -x
+          script/buck2 run //:elm_format -- \
```

Comment:
Would be cool if we could make this run `//:format` and that would just format everything that is to be formatted? Like I guess we also want prettier for md and js.

Reply:
Yes! That’s my vision for it too. There’s very little in the script that’s specific to elm-format. It was a bigger change than makes sense for this PR though!